clients/chain/chain.go

```diff
@@ -0,0 +1,77 @@
+package chain
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
```

Comment:
perhaps this can look as below in order to avoid the ethereum naming part:
```go
const (
	toElrondNameTemplate            = "%sToElrond"
	fromElrondNameTemplate          = "ElrondTo%s"
	baseLogIdTemplate               = "%sElrond-Base"
	elrondDataGetterLogIdTemplate   = "%sElrond-ElrondDataGetter"
	clientLogIdTemplate             = "%sElrond-%sClient" 
	elrondRoleProviderLogIdTemplate = "%sElrond-ElrondRoleProvider"
	chainRoleProviderLogIdTemplate  = "%sElrond-%sRoleProvider"
	broadcasterLogIdTemplate        = "%sElrond-Broadcaster"
)
```
and similar for the exported methods

Reply:
done